### DISCUSSION TITLE
Doubt about industry session

### DISCUSSION THREAD

**Discussion by Himashri567** [github/india#325] [Students]
I watched 2 videos in the industry session without skipping or forwarding or pausing. Still the status for both of the videos are showing incomplete. Will it be updated only after watching all the videos?

**Comment by Sudoo69** [comment]
nah it will not. happened with me too. what i did was i watched again or more like it was being played in the background. im planning on watching it on 2x after evrything is compleleted. i got bugged out 6 times so much waste of data.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
